view/frontend/web/js/autocomplete.js

```diff
@@ -39,7 +43,7 @@ define([
      * Initialise Algolia client
      * Docs: https://www.algolia.com/doc/api-client/getting-started/instantiate-client-index/
      **/
-    const searchClient = algoliaBundle.algoliasearch(
+    const searchClient = algoliaSearch(
```

Comment:
```suggestion
    const searchClient = algoliasearch(
```

Reply:
That sounds good to me. A little more native than camel case.